pkg/postage/batchstore/mock/store_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mock_test
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/ethersphere/bee/pkg/postage/batchstore/mock"
+	postagetesting "github.com/ethersphere/bee/pkg/postage/testing"
+)
+
+func TestBatchStoreGet(t *testing.T) {
```

Comment:
Get and Put tests could be merged into one test

Reply:
Fixed and improved coverage.